README.rst

```diff
@@ -89,12 +90,12 @@ Torchvision currently supports the following image backends:
 **Notes:** ``libpng`` and ``libjpeg`` must be available at compilation time in order to be available. Make sure that it is available on the standard library locations,
 otherwise, add the include and library paths in the environment variables ``TORCHVISION_INCLUDE`` and ``TORCHVISION_LIBRARY``, respectively.
 
-.. _libpng : http://www.libpng.org/pub/png/libpng.html
-.. _Pillow : https://python-pillow.org/
-.. _Pillow-SIMD : https://github.com/uploadcare/pillow-simd
-.. _accimage: https://github.com/pytorch/accimage
-.. _libjpeg: http://ijg.org/
-.. _libjpeg-turbo: https://libjpeg-turbo.org/
+1.) _libpng : http://www.libpng.org/pub/png/libpng.html
+2.) _Pillow : https://python-pillow.org/
+3.) _Pillow-SIMD : https://github.com/uploadcare/pillow-simd
+4.) _accimage: https://github.com/pytorch/accimage
+5.) _libjpeg: http://ijg.org/
+6.) _libjpeg-turbo: https://libjpeg-turbo.org/
```

Comment:
This doesn't seem to render correctly, can you revert this change? 
![image](https://user-images.githubusercontent.com/9110200/94696810-c1890880-0337-11eb-9d84-e6228efa2371.png)